src/gettingstarted/dev-environments/create-environment.md

```diff
@@ -0,0 +1,52 @@
+# Development Environments
+
+## Create environment
+
+Platform.sh supports the deployment of isolated development environments from your branches, complete with exact copies of all of your data. This is useful for testing changes in isolation before merging them.
+
+
+<asciinema-player src="/videos/asciinema/branch.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Create branch, make changes, push to Platform.sh**
+
+    Create and checkout a branch for your new feature.
+
+    ```
+    platform branch dev
+    ```
+
+    This command will create a new branch `dev` from `master`, as well as a local `dev` branch for you to work on. `dev` will be a clone of `master`, including an exact-copy of all of its data and files.
+
+    Make changes to your code, commit them, and push to the Platform.sh remote.
+
+    ```bash
+    git add .
+    git commit -m "Commit message."
+    git push platform dev
```

Comment:
Again, the `platform dev` is unnecessary if an upstream tracking branch has been set.

Reply:
Wouldn't it be necessary if if their repository has multiple remotes (i.e. an origin?). I always thought if there was a default it might default to origin.